superbit_lensing/match.py

```diff
@@ -4,74 +4,104 @@
 
 import numpy as np
 import esutil.htm as htm
-from astropy.table import Table
+from astropy.table import Table, hstack
 
 class MatchedCatalog(object):
-    '''
-    Could have a matched class not dependent on
-    a truth catalog, but haven't had a use yet
```

Comment:
This could be a nice enhancement to add for when we have flight data.

Reply:
I'm unclear if you meant it this way because the comment is on the "old" side of the code, but I refactored these two classes now because I finally had a use for it (e.g. matching SE & ngmix catalog)

Reply:
in other words, it _is_ added in this PR

Reply:
Thank you for clarifying, I see that now. Great!